recipe/meta.yaml

```
@@ -27,7 +27,7 @@ requirements:
     - {{ stdlib('c') }}
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - numpy                                  # [build_platform != target_platform]
+    - numpy >=2                                # [build_platform != target_platform]
```

This isn't needed as it is already handled via [the NumPy 2 migrator]( https://github.com/conda-forge/numcodecs-feedstock/blob/100c42bb227b0ea456c1a348a6cb2a4744928059/.ci_support/migrations/numpy2.yaml ). Let's leave this unpinned here

```suggestion
    - numpy                                  # [build_platform != target_platform]
```

---

Isn't it safer to leave it pinned to match the source package?

---

Not really the migrator is setting it to NumPy 2. When the migrator completes, that will be the default at build time

The only reason we would change this is if we needed to build with something even newer (like NumPy 2.1)